samples/logging_s3/pipeline_log_to_s3_by_fluentd_recommend/fluent-config.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-config
+  namespace: kube-system
+data:
+  fluent.conf: |-
+    <source>
+      @type tail
+      path "/var/log/containers/*.log"
+      pos_file /var/log/fluentd-docker.pos
+      time_format %Y-%m-%dT%H:%M:%S
+      tag kubernetes.*
+      format json
+      read_from_head true
+    </source>
+
+    <filter kubernetes.**>
+      @type kubernetes_metadata
+    </filter>
+
+    <match kubernetes.**>
+      @type rewrite_tag_filter
+      <rule>
+        key $['kubernetes']['labels']['app_kubernetes_io/managed-by']
+        pattern /^.*tekton-pipelines.*$/
+        tag tekton-pipelines
+      </rule>
+    </match>
+
+    <filter tekton-pipelines>
+      @type record_transformer
+      enable_ruby
+      <record>
+        pipelinerun ${record["kubernetes"]["labels"]["tekton_dev/pipelineRun"]}
+        pipelinetask ${record["kubernetes"]["labels"]["tekton_dev/pipelineTask"]}
+      </record>
+    </filter>
+
+    # <match tekton-pipelines>
+    #   @type stdout
+    # </match>
+
+    <match tekton-pipelines>
+      @type s3
+      aws_key_id "#{ENV['AWS_KEY_ID']}"             # variabel of aws_key_id, e.g: admin123
+      aws_sec_key "#{ENV['AWS_SEC_KEY']}"           # variabel of aws_sec_key, e.g: admin123
+      s3_endpoint "#{ENV['S3_ENDPOINT']}"           # variabel of s3_endpoint, if you are using minio, e.g: http://9.21.53.162:31846/
+      force_path_style "#{ENV['FORCE_PATH_STYLE']}" # This prevents AWS SDK from breaking endpoint URL, set true if you are using minio
+      s3_bucket "#{ENV['S3_BUCKET']}"               # variabel of s3_bucket, e.g: mlpipeline
```

Comment:
```suggestion
      s3_bucket "#{ENV['S3_BUCKET']}"               # variable of s3_bucket, e.g: mlpipeline
```

Reply:
Oh，it's my spelling mistakes，Thanks @Tomcli , modifyed it.  
Mark as resolved, will look into  #276.